security/libprelude/Portfile

```diff
@@ -3,47 +3,55 @@
 PortSystem      1.0
 
 name            libprelude
-version         1.0.1
-set download_id 241
-revision        5
+version         5.1.1
+revision        1
 categories      security
 license         GPL-2+
-maintainers     nomaintainer
+maintainers     gmail.com:thomas.andrejak
```

Comment:
please also add your GitHub handle and, if possible, `openmaintainer` so that others can make minor changes without needing your permission. The line could become: `maintainers     {gmail.com:thomas.andrejak @ToToL} openmaintainer`

Reply:
ok, done